src/main/java/com/byndyusoft/template/client/TemplateClient.java

```diff
@@ -0,0 +1,9 @@
+package com.byndyusoft.template.client;
+
+import com.byndyusoft.template.client.dto.TemplateRequestDto;
+import com.byndyusoft.template.client.dto.TemplateResponseDto;
+
+public interface TemplateClient {
+  TemplateResponseDto getTemplateMethod(String uriParam);
```

Comment:
I suggest adding `throws RestClientException` to show exactly that we are not handling this error, but forwarding it, and it needs to be handled.

```suggestion
  TemplateResponseDto getTemplateMethod(String uriParam) throws RestClientException;
```

Reply:
done